src/views/Links/index.jsx

```diff
@@ -2,7 +2,9 @@ import { Card, CardContent, CardHeader, Grid, Typography, Box, Button } from '@m
 import LinkIcon from '@mui/icons-material/InsertLink';
 import MapOutlinedIcon from '@mui/icons-material/MapOutlined';
 import NotificationsNoneOutlinedIcon from '@mui/icons-material/NotificationsNoneOutlined';
-import { ListItemIcon, ListItemText, List, ListItem, Link } from '@mui/material';
+import { ListItemIcon, ListItemText, List, ListItem } from '@mui/material';
+import { Link as RouterLink } from 'react-router-dom';
+import { Link as MuiLink } from '@mui/material';
```

Comment:
Can you just leave the `MuiLink` as just `Link` since the `react-router-dom` Link is named something else?